### ISSUE TITLE
Add `Disable Super key override` option back to advanced preferences

### ISSUE THREAD

**Issue by jacobgkau** [pop-os/gnome-shell-extension-pop-cosmic#222]
Seems to have gotten lost in the 21.10 version, but is still present in 21.04.

**Comment by Glint-Eye** [comment]
Yes, please!

**Comment by Immedicale** [comment]
Definitely needed!

**Comment by Rick1029** [comment]
Definitely needed. Can't use arcmenu very well anymore since Pop opens the applications menu instead.

I believe this was removed because of #195 but in doing so it just breaks other extension functionality. Disabling the cosmic extension fixes this (but you lose cosmic in the process.) Workspaces still work as it is a separate extension so not a massive deal breaker.

**Comment by jacobgkau** [comment]
> I believe this was removed because of #195

That is incorrect, this was not removed because of #195; that issue was fixed independently on Pop!_OS 21.04. It was just lost during the work to update COSMIC to support GNOME 40 for Pop!_OS 21.10.
